### PR TITLE
Misc documentation improvements

### PR DESCRIPTION
**Pull request by sternenseemann**
Includes the requested HTML improvements (UTF-8 works!).

Github Pages publishing lookis [like this](https://sternenseemann.github.io/ocaml-emoji/), the branch name can be also be changed and checked out somewhere else (e. g. your webspace) without having the generated documentation in the main branch.

### CODE REVIEW

**Comment by fxfactorial** [comment]
Awesome! But you worked too hard on the github shell script. With github now you can just make a directory called `docs` on master, version control that and it shows up on your `githubname.github.io/<project-name>`

No more separate github pages branches are needed, makes it simpler.

File: _oasis [hunk 1]
```diff
@@ -14,16 +14,20 @@ BuildTools:    ocamlbuild, ocamldoc
 Description:   Names to byte code sequences for common emojis
 
 Library emoji
-  Path: lib
+  Path:    lib
```
Comment (fxfactorial) [inline _oasis:17]: small annoyance, why this white space?

Comment (sternenseemann) [reply _oasis:17]: alignment

**Comment by sternenseemann** [comment]
Oh really? I still feel like it is cleaner if the built documentation is separated from the source code, but that's just my preference. Should I just import it into master?

**Comment by fxfactorial** [comment]
Yea I usually would agree but the ease of GitHub, only using one branch is imo easier. Up to you what you think is better and we go with that.

**Comment by sternenseemann** [comment]
Both would be easy for me, just tell me, what you prefer!

**Comment by fxfactorial** [comment]
I prefer the simpler one with less code, which is a good rule of thumb I think :)

**Comment by sternenseemann** [comment]
@fxfactorial there you go :)

**Comment by fxfactorial** [comment]
@sternenseemann Perfect, thanks! accepted, looks good: http://hyegar.com/ocaml-emoji/Emoji.html 

Cutting a new release for opam now

**Comment by sternenseemann** [comment]
\o/

**Pull request merged**